bindata/network/ovn-kubernetes/ovnkube-node.yaml

```diff
@@ -46,12 +46,17 @@ spec:
             set -o allexport
             source "/env/${K8S_NODE}"
             set +o allexport
-          fi
+          fi  
+          
           echo "$(date -Iseconds) - starting ovn-controller"
           exec ovn-controller unix:/var/run/openvswitch/db.sock -vfile:off \
             --no-chdir --pidfile=/var/run/ovn/ovn-controller.pid \
+            --syslog-method="{{.OVNPolicyAuditDestination}}" \
+            --log-file=/var/log/ovn/acl-audit-log.log \
+            -vFACILITY:"{{.OVNPolicyAuditSyslogFacility}}" \
             -p /ovn-cert/tls.key -c /ovn-cert/tls.crt -C /ovn-ca/ca-bundle.crt \
-            -vconsole:"${OVN_LOG_LEVEL}"
+            -vconsole:"${OVN_LOG_LEVEL}" -vconsole:"acl_log:off" \
+            -vfile:"acl_log:${OVN_LOG_LEVEL}" -vsyslog:"acl_log:${OVN_LOG_LEVEL}"
```

Comment:
I was surprised to see both the file and syslog configs here.  why both?

Reply:
I now explicitly configure all of the ovs logging modules here 
1. Before the `ovnkube-node` container dumped all controller logs now it dumps all logs except the ACL audit logs to stdout 
2. The file output only contains ACL audit logging which is stored on node and also tailed by the `acl-audit-logging` sidecar container 
3. The feature can be configured to also send ACL audit logs to the underlying Host's journald process or over a udp socket which is when the syslog module is configured here.